OsuPlayer/Views/PlaylistView.axaml.cs

```diff
@@ -60,4 +62,15 @@ private async void PlayPlaylist_OnClick(object? sender, RoutedEventArgs e)
 
         await ViewModel.Player.TryPlaySongAsync(ViewModel.Player.SongSourceProvider.GetMapEntryFromHash(playlist.Songs.First()));
     }
+
+    private void PlaylistItemLoaded(object? sender, VisualTreeAttachmentEventArgs e)
+    {
+        if (sender is not Grid grid) return;
```

Comment:
I like this for some reason, didn't know this is even a thing

Reply:
Yes good features :)